### PR TITLE
Support deserialization of Any type

### PR DESCRIPTION
**Pull request by jdechicchis**

### CODE REVIEW

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java [hunk 1]
```diff
@@ -43,19 +43,7 @@ public BufSchemaRegistryClient(ManagedChannelBuilder<?> channelBuilder, String a
...
-      log.warn("Cannot get package name and type name from {}", fullyQualifiedTypeName);
-      return Optional.empty();
-    }
-
-    String packageName = String.join(".", parts.subList(0, parts.size() - 1));
-    String typeName = parts.get(parts.size() - 1);
-
-    log.info("Looking for type {} in package {}", typeName, packageName);
-
+  public Optional<List<FileDescriptor>> getFileDescriptors(String owner, String repo) {
```
Comment (csueiras) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java]: would highly recommend simply returning an empty list, `Optional<List<T>>` vs empty list is probably exactly the same outcome?

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java [hunk 2]
```diff
@@ -82,17 +70,38 @@ public BufSchemaRegistryClient(ManagedChannelBuilder<?> channelBuilder, String a
...
+    String typeName = parts.get(parts.size() - 1);
+
+    log.info("Looking for type {} in package {}", typeName, packageName);
+
+    Optional<List<FileDescriptor>> fileDescriptors = getFileDescriptors(owner, repo);
+    if (fileDescriptors.isEmpty()) {
+      return Optional.empty();
+    }
+
+    return Optional.ofNullable(fileDescriptors.get()
```
Comment (csueiras) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java]: You can map an optional into another

```java
fileDescriptors.stream()
  .flatMap(d -> d.stream())
  .filter(...)
```

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java (hunk 1 above)
Comment (jdechicchis) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java]: fixed

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java (hunk 2 above)
Comment (jdechicchis) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/client/BufSchemaRegistryClient.java]: good to know. updated the return value so this no longer applies for this case

**Comment by csueiras** [comment]
+1

**Pull request merged**